src/collection-view.js

```diff
@@ -76,7 +76,7 @@ const CollectionView = Backbone.View.extend({
   },
 
   _endBuffering() {
-    const shouldTriggerAttach = !!this._isAttached;
+    const shouldTriggerAttach = !!this._isAttached && this.monitorViewEvents !== false;
```

Comment:
I suppose so, but I didn't add them.  There are a lot of other cases of `!!` casting in similar places, perhaps a separate PR that investigated and changed that would be better than just in the instances?